legal-api/report-templates/template-parts/certification.html

```diff
@@ -1,6 +1,6 @@
         <div  class="no-page-break">
           <div class="section-title">
-            CERTIFICATION
+            Certification
```

Comment:
Changed per latest designs. (Yes, it affects the other filings too... probably a good thing.)